mapswipe-dashboard/app/views/UserGroupDashboard/index.tsx

```diff
@@ -104,6 +124,17 @@ function UserGroupDashboard(props: Props) {
         { member: item }
     ), []);
 
+    const handleMembersDownload = useCallback(() => {
+        const headers = ['User', 'Total swipes', 'Mission contributed', 'Time spent(mins)'].join(',');
+        const members = userGroupStats?.userGroup.userStats?.map(
+            (user) => (
+                [user.userName, user.totalSwipes, user.totalMappingProjects, user.totalSwipeTime].join(',')
+            ),
+        );
+        const data = [headers, ...(members ?? [])].join('\n');
+        downloadFile(data, `${userGroupStats?.userGroup.name}-members`, 'text/csv');
```

Comment:
We should use a csv library so that we do not have to deal with escaping characters

Reply:
used react-csv